Service/ImageService.cs

```diff
@@ -18,5 +22,31 @@ public string ImageBase64(byte[] imageData)
 
             return $"data:image/png;base64,{Convert.ToBase64String(imageData)}";
         }
+
+        public async Task<List<Image>> UploadImagesAsync(List<IBrowserFile> files, string moveName)
+        {
```

Comment:
Jest względnie Ok, ale możesz to zrefaktorować - nie musić robić na ReadStream podczas MemoryStream - ale to pierdoła akurat